### PR TITLE
enhancement: add AMA to citation styles

### PR DESCRIPTION
**Pull request by qweliant**
add AMA citations to the options citation style

### CODE REVIEW

**Review by gabestein — APPROVED** [review]
LGTM! 

Tiny thing: we could probably just get away with "AMA." I don't think they release new additions very often, and the 11th is from 2020. We have APA 7th only because it is very recent and so some people are still using 6th.

File: utils/citations.ts [hunk 1]
```diff
@@ -44,6 +44,7 @@ export const citationStyles: CitationStyle[] = [
 	{ key: 'elife', name: 'ELife', path: './citeStyles/elife.csl' },
 	{ key: 'frontiers', name: 'Frontiers', path: './citeStyles/frontiers.csl' },
 	{ key: 'mla', name: 'MLA', path: './citeStyles/modern-language-association.csl' },
+	{ key: 'apa', name: 'AMA 11th edition', path: './citeStyles/american-medical-association.csl' },
```
Comment (gabestein) [inline utils/citations.ts:47]: should be `ama`, not `apa`.

**Review by gabestein — CHANGES_REQUESTED** [review]
Wait I take it back, sorry. Tiny little language issue.

**Review by gabestein — CHANGES_REQUESTED** [review]
Sorry, one more thing I added late to the other review: `CitationStyleKind` will need to be updated to include ama.

**Comment by qweliant** [comment]
> Sorry, one more thing I added late to the other review: `CitationStyleKind` will need to be updated to include ama.

didnt see this before i re-requested, i will change it

**Review by gabestein — APPROVED** [review]
👍 for real this time! Thanks!

**Pull request merged**